recipes-graphics/wayland/weston-init/weston-waitforcard.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+DEFAULT_CARD=/dev/dri/card0
```

Comment:
This patch takes into account that only DRM backend is used. Weston can use different backends, like FBDev.
If we want to limit the functionality to FBDev, that needs to be also reflected in the Weston recipes.